week2/4_Lists.ipynb

```diff
@@ -1016,7 +1017,7 @@
     "<ol style=\"text-align: right; direction: rtl; float: right; clear: both;\">\n",
     "    <li>הביטוי הבוליאני בשורה 1 מחזיר <samp>False</samp>, מכיוון שכל אחד מהאיברים ברשימה <var>pupils</var> הוא רשימה, ואף אחד מהם אינו המחרוזת <em>\"Moshe\"</em>.</li>\n",
     "    <li>הביטוי הבוליאני בשורה 2 מחזיר <samp>True</samp>, מכיוון שהרשימה האחרונה בתוך <var>pupils</var> מחזיר רשימה המכילה את המחרוזת <em>\"Moshe\"</em>.</li>\n",
-    "    <li>הביטוי הבוליאני בשורה 3 מחזיר <samp>False</samp>, מכיוון אשאין בתוך <var>pupils</var> רשימה שאלו בדיוק הערכים שלה. יש אמנם רשימה שמכילה את האיברים האלו, אבל השאלה הייתה האם הרשימה הגדולה (<var>pupils</var>) מכילה ישירות את הרשימה הקטנה (<code>['Moshe', 'Splinter']</code>).</li>\n",
+    "    <li>הביטוי הבוליאני בשורה 3 מחזיר <samp>False</samp>, מכיוון שאין בתוך <var>pupils</var> רשימה שאלו בדיוק הערכים שלה. יש אמנם רשימה שמכילה את האיברים האלו, אבל השאלה הייתה האם הרשימה הגדולה (<var>pupils</var>) מכילה איבר שהוא שווה בדיוק ל- (<code>['Moshe', 'Splinter']</code>).</li>\n",
```

Comment:
שהוא שווה -> ששווה
Use less verbose form when possible.
Also, change the - to ־